src/main/resources/application.properties

```diff
@@ -55,6 +55,7 @@ LOGGING_INFO_ENABLE=false
 INTERNAL_TOPIC=internal
 NATS_MAX_AGE=86400000
 NATS_MAX_SIZE=8388608
+NATS_MAX_CLIENT_ID=50
```

Comment:
Return the value in `GET /config` so Client GW can reference this.

Reply:
ok

Reply:
updated